### PR TITLE
Update Universal Search fixture data.

### LINKED ISSUE #571: Update the version of UniversalSearch XPI that's currently on dev
Apparently the version being served by testpilot.dev.mozaws.net is a couple months old, so we should check the XPI path to make sure it's pointing to the latest version.

### PR DESCRIPTION
**Pull request by chuckharmston**
Closes #571 

r? @pdehaan

### CODE REVIEW

File: fixtures/demo_data.json [hunk 1]
```diff
@@ -191,8 +191,8 @@
     "thumbnail": "demo/experiments_experiment/a/8/a8182b008f6434ba702ab177232625e2_thumbnail_1449675941_0564.png",
     "created": "2015-12-09T15:45:42.395Z",
     "slug": "universal-search",
-    "xpi_url": "https://people.mozilla.org/~jhirsch/universal-search-addon/addon.xpi",
-    "contribute_url": "https://github.com/mozilla/universal-search-addon/blob/master/CONTRIBUTING.md"
+    "xpi_url": "https://s3-us-west-2.amazonaws.com/universal-search/universal-search-1.0.0b1.xpi",
```
Comment (pdehaan) [inline fixtures/demo_data.json]: Why a specific filename vs always just renaming to "universal-search.xpi" or "latest.xpi"?

**Comment by lmorchard** [comment]
FWIW, this will not actually fix #571 - dev is not auto-updated with this fixture data. It's a manual Django loaddata operation. 

This should probably also be rolled into #555, because I think @ckprice is currently updating the fixture overall.

File: fixtures/demo_data.json (hunk 1 above)
Comment (lmorchard) [inline fixtures/demo_data.json]: Yeah, you might want to use a consistent URL pointing to the latest, so we don't need to update all the URLs with every release

Comment (chuckharmston) [inline fixtures/demo_data.json]: Yep, that was my bad. Copy/pasted from the wrong thing during a meeting.

Comment (pdehaan) [inline fixtures/demo_data.json]: But note, we'll still need to update the version number on every release manually through the /admin/ (and possibly in the fixtures/demo_data.json) for consistency maybe.

**Comment by lmorchard** [comment]
#571 was manually addressed, but will merge this to update data for local installs. We should make sure it gets carried over in #555

**Comment by pdehaan** [comment]
**NOTE:** I also tweaked the data on the admin site on http://testpilot.dev.mozaws.net/experiments/universal-search to reflect the latest changes.

**Pull request merged**